kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/storage/ControllerStorageFileWatcher.java

```diff
@@ -35,12 +30,16 @@
 public class ControllerStorageFileWatcher implements Runnable {
     
     private static final Logger log = LoggerFactory.getLogger(ControllerStorageFileWatcher.class);
+    public static final String STORAGE_FILE_WATCHER_INTERVAL = "org.kie.server.controller.templatefile.watcher.interval";
```

Comment:
Just a nitpick, please move this constant to KieServerConstants to have them all in one place

Reply:
let's first see if this works and then we can fine tune it :)